src/app/(app)/(public)/features/page.tsx

```
@@ -0,0 +1,374 @@
+              <div className="grid lg:grid-cols-2 gap-12 items-center">
+                <div className="space-y-6">
+                  <Badge variant="outline">Seamless Integration</Badge>
+                  <h3 className="text-3xl font-bold text-foreground">
+                    Connect Everything
+                  </h3>
+                  <p className="text-lg text-muted-foreground">
+                    Integrate with your favorite tools and services. Our extensive API and pre-built connectors make it easy to create a unified workflow across all your platforms.
+                  </p>
+                  <ul className="space-y-3">
+                    {["REST API access", "Webhook support", "Pre-built connectors", "Custom integrations"].map((item, index) => (
+                      <li key={index} className="flex items-center">
```

Biome (2.1.2)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)